projects/storefrontapp/src/styles.scss

```diff
@@ -1,3 +1,17 @@
 // Spartacus Application styles
 
 @import 'storefrontstyles/index';
+
+cx-product-facet-navigation {
+  button.dialog-trigger {
```

Comment:
Why did you need put this here?

Reply:
Just to demo that we can influence the experience to force the dialog by showing the button.